beginner_source/quickstart/tensor_tutorial.py

```diff
@@ -0,0 +1,263 @@
+print('Reshaped tensor:\n',x.view(5, -1))  # will result in size 5x3
+
+
+######################################################################
+# The number of elements in a view should be the same as in the
+# original tensor. You can use ``-1`` in one of the dimensions to
+# figure out this dimension automatically.
+#
+
+
+######################################################################
+# .. note:: ``view`` is similar to ``reshape`` operation in NumPy. There
+#           is also a ``reshape`` method available in PyTorch, and it is more
+#           powerful than ``view``, because it can also reshape non-contiguous
+#           arrays by copying them to the new shape. However, in vast majority of
+#           cases you can use ``view`` and make sure that no data copying occurs,
+#           and the operation is always efficient.
+#
+
+
+######################################################################
+# Tensors support all slicing operations that exist in NumPy:
+#
+
+print(x.size())  # original size of x is 3x5
+print('First row: ',x[0])
+print('First column: ', x[:, 0])
+print('Last column:', x[..., -1])
+
+
+######################################################################
+# If you have a one-element tensor, for example, after aggregating all
+# values of the tensor into one value, you can convert it to a Python
+# numerical value using ``item()``:
+#
+
+val = x.sum().item()  # will compute the sum of all elements
+print(val)
+
+######################################################################
+# Hardware-Accelerated Computations
+# ~~~~~~~~~~~~~~~~
+#
+# One of the major benefits of using PyTorch is the ability to perform
+# tensor operations on GPUs and some other specialized hardware. To do that,
+# we need to explicitly **move** tensors to another computing platform using ``.to`` method.
+#
+# In most of the cases, we check for the availability of GPU in the beginning
+# of the script, and define the ``device`` object accordingly. Then we move all
+# tensors to that device before performing the computations:
+#
+
+if torch.cuda.is_available():
+    device = torch.device("cuda")
+else:
+    device = torch.device("cpu")
+
+print("Doing computations on {}".format(device))
+
+x = torch.randn(3, 5, device=device)  # create tensor on specified device
+y = torch.ones_like(x)  # create tensor on CPU
+y = y.to(device)  # move tensor to another device
+z = x+y  # this is performed on GPU if it is available
+print(z.to("cpu", torch.double))
+
+
+######################################################################
+# In the last operation, when we move the tensor back to the CPU, we can
+# also change the ``dtype``. This does not result in additional
+# computational time, because we need to copy and transform the data when
+# moving it from GPU anyway.
+#
+# Next learn how to load built in and custom `datasets with dataloaders <dataquickstart_tutorial.html>`_
```

Comment:
@cassieview I've pinged you on slack about this - can we replace these with "<Previous" and "Next>" links like in the footer of `quickstart_tutorial.py`

Reply:
@suraj813 this is a good idea. I will make this change!